ihatemoney/templates/list_bills.html

```diff
@@ -106,31 +108,50 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
-                    title="{{ _('Added on %(date)s', date=bill.creation_date if bill.creation_date else bill.date) }}">
-                    {{ bill.date }}
-                </span>
-            </td>
-            <td>{{ bill.payer }}</td>
-            <td>{{ bill.what }}</td>
-            <td>{{ bill.owers|join(', ', 'name') }} </td>
-            <td>{{ "%0.2f"|format(bill.amount) }} ({{ "%0.2f"|format(bill.pay_each()) }} {{ _("each") }})</td>
-            <td class="bill-actions">
-                <a class="edit" href="{{ url_for(".edit_bill", bill_id=bill.id) }}" title="{{ _("edit") }}">{{ _('edit') }}</a>
-                <a class="delete" href="{{ url_for(".delete_bill", bill_id=bill.id) }}" title="{{ _("delete") }}">{{ _('delete') }}</a>
-            </td>
-    </tr>
-    {% endfor %}
-    </tbody>
-    </table>
+        <div class="clearfix"></div>
+
+        <table id="bill_table" class="col table table-striped table-hover table-responsive-sm">
+            <thead><tr><th>{{ _("When?") }}</th><th>{{ _("Who paid?") }}</<th><th>{{ _("For what?") }}</th><th>{{ _("For whom?") }}</th><th>{{ _("How much?") }}</th><th>{{ _("Actions") }}</th></tr></thead>
+        <tbody>
+        {% for bill in bills %}
+        <tr owers="{{bill.owers|join(',','id')}}" payer="{{bill.payer.id}}">
+                <td>
+                    <span data-toggle="tooltip" data-placement="top"
+                        title="{{ _('Added on %(date)s', date=bill.creation_date if bill.creation_date else bill.date) }}">
+                        {{ bill.date }}
+                    </span>
+                </td>
+                <td>{{ bill.payer }}</td>
+                <td>{{ bill.what }}</td>
+                <td>{{ bill.owers|join(', ', 'name') }} </td>
+                <td>{{ "%0.2f"|format(bill.amount) }} ({{ "%0.2f"|format(bill.pay_each()) }} {{ _("each") }})</td>
+                <td class="bill-actions">
+                    <a class="edit" href="{{ url_for(".edit_bill", bill_id=bill.id) }}" title="{{ _("edit") }}">{{ _('edit') }}</a>
+                    <a class="delete" href="{{ url_for(".delete_bill", bill_id=bill.id) }}" title="{{ _("delete") }}">{{ _('delete') }}</a>
+                </td>
+        </tr>
+        {% endfor %}
+        </tbody>
+        </table>
 
     {% else %}
-    <div class="py-3"></div><p>{{ _("Nothing to list yet. You probably want to") }} <a href="{{ url_for(".add_bill") }}" data-toggle="modal" data-target="#bill-form">{{ _("add a bill") }}</a> ?</p></div>
+        <div class="py-3 d-flex justify-content-center empty-bill">
+        <div class="card d-inline-flex p-2">
+            <div class="card-body text-center text-muted">
+                <i class="icon icon-white hand-holding-heart">{{ static_include("images/hand-holding-heart.svg") | safe }}</i>
+                <h3>{{ _('No bills')}}</h3>
+                <p>
+                    {{ _("Nothing to list yet.")}} <br />
+                    {{ _(" You probably want to") }}
```

Comment:
The space at the beginning of the sentence is here on purpose?

Reply:
Oups no...